apps/explorer/src/ui/TableCard.tsx

```diff
@@ -167,9 +113,12 @@ function TableCard({
                 </thead>
                 <tbody>
                     {table.getRowModel().rows.map((row: any) => (
-                        <tr key={row.id}>
+                        <tr key={row.id} className={'group'}>
                             {row.getVisibleCells().map((cell: any) => (
-                                <td key={cell.id}>
+                                <td
+                                    key={cell.id}
+                                    className={cellStyle({ variant: 'td' })}
```

Comment:
Given these variants aren't really exposed to the user, you could also just style these conditionally with `clsx`. CVA is really about prop-driven styling.

Reply:
Not sure how you want that to work. Does not look necessary.